modules/custom/openy_activity_finder/src/Controller/ActivityFinderController.php

```diff
@@ -99,6 +99,10 @@ public function getData(Request $request) {
     else {
       $data = $this->backend->runProgramSearch($parameters, $log->id());
 
+      /* @var $config_settings \Drupal\Core\Config\Config */
+      $config_settings = \Drupal::service('config.factory')->getEditable('openy_activity_finder.settings');
```

Comment:
This config is already loaded here: https://github.com/ymcatwincities/openy/blob/8.x-2.x/modules/custom/openy_activity_finder/src/Controller/ActivityFinderController.php#L53
Replace this code with $this->config instead.